smdatatools.py

```diff
@@ -104,24 +111,52 @@
     end_time = time.time()
     print("Elapsed data processing time was %g seconds" % (end_time - start_time))
 
+
+
     if hasattr(args, 'copyaudio'):
         print("Copying audio files (if it exists) related to parsed data")
 
         extensions = {'.ogg', '.mp3', '.wav'}
-        audio_filepaths = []
-        audio_filepaths.extend(getFilePaths(dir_sm_input, extensions))
-        audio_filepaths.extend(getFilePaths(dir_txt_input, extensions))
-
-        # create a dictionary pairing the formatted audio filename with their original path
-        audio_name_paths = dict(zip([strip_filename(audio) for audio in audio_filepaths], range(len(audio_filepaths))))
-
-        # copy audio file if it matches any parsed data
-        for data in dataList:
-            if data.filename in audio_name_paths.keys():
-
-                in_audio_path = audio_filepaths[audio_name_paths[data.filename]]
-                out_audio_path = join(dir_output_audio, data.filename + splitext(in_audio_path)[1]).replace("\\","/")
 
-                if isfile(in_audio_path) and not isfile(out_audio_path):
-                    print('  - Copying audio from %s to %s' % (in_audio_path, out_audio_path))
-                    copyfile(in_audio_path, out_audio_path)
+        if args.inputaudio: # forces conversion without needing corresponding sm/txt data
+            audio_filepaths = getFilePaths(args.inputaudio, extensions)
```

Comment:
There should be a standard to using either Camel or snake_case throughout the directory. Since it looks like most of the functions default to using snake_case, this should be updated in the future.

I suggest creating a bug to normalize the naming conventions throughout the file.